core/src/replay_stage.rs

```diff
@@ -620,20 +610,19 @@ mod test {
     use crate::blocktree::{create_new_tmp_ledger, get_tmp_ledger_path};
     use crate::cluster_info::{ClusterInfo, Node};
     use crate::entry::create_ticks;
-    use crate::entry::{next_entry_mut, Entry};
     use crate::fullnode::new_banks_from_blocktree;
     use crate::packet::Blob;
     use crate::replay_stage::ReplayStage;
-    use crate::result::Error;
     use solana_sdk::genesis_block::GenesisBlock;
     use solana_sdk::hash::Hash;
     use solana_sdk::signature::{Keypair, KeypairUtil};
     use solana_vote_api::vote_state::Vote;
     use std::fs::remove_dir_all;
-    use std::sync::mpsc::channel;
     use std::sync::{Arc, RwLock};
 
+    //TODO(sagar-solana) there's no way to make this work as is. Need to be making roots in replay stage
```

Comment:
@sakridge, this test fails because no roots are being made (need to simulate votes and stuff too if we want to make this work the way its written right now).

Reply:
I see. yea I don't really have any good ideas, maybe we would have to add the votes.

Reply:
Or remove the recv on the root_slot_receiver and rely on higher-level integration tests like the local cluster tests.

Reply:
Yeah I think I prefer that. I'll remove it.